test/e2e/storage/persistent_volumes.go

```diff
@@ -326,6 +326,12 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 			ginkgo.It("should be reschedulable [Slow]", func() {
 				// Only run on providers with default storageclass
 				e2eskipper.SkipUnlessProviderIs("openstack", "gce", "gke", "vsphere", "azure")
+				// testsuites/multivolume tests can now run with windows nodes
+				// This test is not compatible with windows because the default StorageClass
+				// doesn't have the ntfs parameter, we can't change the status of the cluster
+				// to add a StorageClass that's compatible with windows which is also the
+				// default StorageClass
+				e2eskipper.SkipIfNodeOSDistroIs("windows")
```

Comment:
There are multiple ways that we can skip tests for Windows. There's [LinuxOnly] tag and [Feature: Windows] tags. Can we be consistent?

Should all default storageclass tests be skipped for windows?

Reply:
There's only one default storageclass test in this file

- The CI config for this test is: https://github.com/kubernetes/test-infra/blob/c73258309a5012dbf9a7540c1b0dcabb0130e7bd/config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml#L226-L271
- Which leads to https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/run-k8s-windows-migration.sh
- The script eventually calls kubetest like this:

```
[kubetest --test --ginkgo-parallel --check-version-skew=false --test_args=--ginkgo.focus=PersistentVolumes\sGCEPD|[V|v]olume\sexpand|\[sig-storage\]\sIn-tree\sVolumes\s\[Driver:\swindows-gcepd\]|allowedTopologies|Pod\sDisks|PersistentVolumes\sDefault --ginkgo.skip=\[Disruptive\]|\[Serial\]|\[LinuxOnly\] --node-os-distro=windows --allowed-not-ready-nodes=3 --storage.migratedPlugins=kubernetes.io/gce-pd  --dump=/logs/artifacts]
```

Great, I added the `[LinuxOnly]` annotation instead